index.js

```diff
@@ -4,8 +4,15 @@ const express = require('express'),
       session = require('express-session'),
       displayRoutes = require('express-routemap'),
       bcrypt = require('bcrypt'),
+      crypto = require('crypto'),
```

Comment:
what is difference between  bcrypt and crypto packages? where can read more about them?